lib/Conch/Controller/DeviceReport.pm

```diff
@@ -282,7 +285,7 @@ sub _record_device_report {
 
 				my $mac = uc( $dr->{interfaces}->{$nic}->{mac} );
 
-				$log->debug("Device $device_id: Recording NIC: $mac");
+				$log->debug("Device $device->id: Recording NIC: $mac");
```

Comment:
This will not render correctly. use `$device_id`.

Reply:
Addressed